docs/src/administrator/manual/django-admin.md

```diff
@@ -12,72 +12,69 @@ _![Django Admin](./img/admin-1.png)_
-4. **Save and continue editing**: Save current record while still showing current record.
-5. **Save**: Save current record, then redirect to Django Admin Table/record list.
-6. **History**: Button to see actions applied to current record.
+    4. **Change My Password Button**: Once users have filled in the required information, they can click the `CHANGE MY PASSWORD` button to submit the form. If all the input is valid and the new password meets the criteria, the password for the user account will be updated.
 
-## Activities Table
+After successfully changing the password, the user will receive a confirmation message indicating that the password has been updated.
 
-_![Activities Table](./img/admin-5.png)_
+4. **Log Out**: Clicking on this option will log the user out of the admin panel and return the user to the login page.
 
-This table is used to store existing activity types e.g. `Unplanned/natural deaths`.
+    ![Logout admin page](./img/admin-9.png)
 
-1. **Colour**: Used as color identifier in reports and charts. The default is black (`#000000`).
-2. **Width**: Column width in the Activity Report.
-3. **Export fields**: Used as export fields in Activity Report. The value should be an array/list, containing
-`Annual Population Per Activity` field to export. Currently available fields are:
-    * founder_population
-    * reintroduction_source
-    * intake_permit
-    * offtake_permit
-    * translocation_destination
+    1. **Log in again**: Clicking this option will open the login page.
 
-This is an example of correct **Export fields** value:
-`["translocation_destination", "founder_population"]`
+5. **Recent Actions**: The `Recent Actions` section is typically located on the right side of the admin panel and provides a list of recent actions that have been performed within the admin interface. These actions may include additions, deletions, or changes to data in your project. Clicking on any of the available links will take a user to the detail of recent action.
 
-Any update on this table will be reflected on the [Activity Report](../../user/manual/explore/reports.md).
+6. **Site Administration**: Site Administration this section provides links to various models and apps registered with the admin panel. It serves as the primary navigation point for managing a project's data. Clicking on any of the available links will take a user to the respective model's management page.
 
-## Group Table
+7. **Add**: This button is used to create a new item of the associated model. Clicking on `Add` link will open a form where users can enter the details of the new record.
+
+8. **Change**: This button is used to edit an existing item from the list. When users click the `Change` button, they will be redirected to a page where they can see all the associate data of that model and can select data to change.
+
+* **Table**: To see data in a table, click on the table name.
```

Comment:
Is this line still necessary? The same information is in line 72

Reply:
as we are doing the doc in tutorial style I think that having this line is okay

Reply:
This is not a tutorial style document, this is the manual which is a page-by-page explanation. Please remove this line as the information is repeated in line 72 and it is more logical for it to be in line 72.

Reply:
Ah sorry, yes it is manual documentation. I will remove this line.